### PR TITLE
French traduction

### PR DESCRIPTION
**Pull request by Atala**
Hi,

Here are the files for the french translation of Arguman!

It's kind of OK but still a work in progress, as we are waiting for feedback on the logical terms translation. Maybe I will ask you for some explanations, as english terms don't always have the exact french counterpart.

I think I will go ahead and deploy my own instance of Arguman.

Thanks for the amazing & neat project!

Kudos to @silvangiraud for his help.

(sorry for the ugly PR, I didn't know the github comments will show up)

### CODE REVIEW

File: web/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -252,7 +252,7 @@ msgstr "Notification par email"
 #: templates/fallacies.html:29 templates/newsfeed.html:39
 #: templates/premises/contention_list.html:67
 msgid "Load more"
-msgstr "Daha fazla yükle"
+msgstr "Montrer plus"
```
Comment (EHadoux) [inline web/locale/fr/LC_MESSAGES/django.po]: "Voir plus" plutôt non ?

**Comment by fatiherikli** [comment]
Thank you for your contributions. We are going to test on our local computers and deploy on the production soon as soon 👍

**Comment by Atala** [comment]
Thanks @fatiherikli - I have been quite busy lately so I didn't have time to check the display on my local machine - please ping me back if any text is too long or break display.

**Comment by fatiherikli** [comment]
screen shot 2016-05-13 at 15 37 02

Could we shorten this text? Do you have any suggestion? @Atala

**Comment by fatiherikli** [comment]
Can we translate also about page and mention french i18n contributors?

**Comment by fatiherikli** [comment]
You can send me just a md file, I can upload it. 

Here is the english version: 
https://raw.githubusercontent.com/arguman/arguman.org/master/web/templates/about-en.md

**Comment by EHadoux** [comment]
I would personally go for:
"arguman.org est une plateforme (note the E) d'argumentation"

**Comment by fatiherikli** [comment]
screen shot 2016-05-13 at 15 55 18

Just a single word too :)

**Comment by fatiherikli** [comment]
And how can I link `analysis`?

Here is the original string:

    "arguman.org is an <a href=\"%(about_url)s\">argument analysis</a> platform."

**Comment by fatiherikli** [comment]
Is that correct?

screen shot 2016-05-13 at 16 28 04

**Comment by silvangiraud** [comment]
It's correct, but we lose the "analytical" aspect with this translation.

**Comment by fatiherikli** [comment]
It's deployed now.
http://fr.arguman.org/

Could you submit a few arguments for the announcement on twitter?

**Pull request merged**